src/app.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// Write your code here
+const readline = require('readline');
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput');
+const { generateRandomNumber } = require('./modules/generateRandomNumber');
+const { getBullsAndCows } = require('./modules/getBullsAndCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let userInput = null;
+
+async function askForNumber() {
+  while (true) {
```

Comment:
Do we need to introduce a `while(true)` loop here? Let think of a possible refactor here.